package.json

```diff
@@ -3,6 +3,9 @@
   "description": "A collection of experimental web components and tools for building Brightspace applications.",
   "type": "module",
   "repository": "https://github.com/BrightspaceUI/labs.git",
+  "exports": {
+    "./controllers/computed-values.js": "./src/controllers/computed-values/index.js"
+  },
```

Comment:
Down below in the `files` section you can now remove `!demo` and `!test` and `!screenshots` since those aren't nested under `src` anymore. I'm not totally sure how `files` and `exports` play together in terms, but we'll see what happens when this publishes! Might need to adjust after.

Reply:
Ah, yes. Forgot about those. Updated now. 👍